site/app/templates/course/UploadCourseMaterialsForm.twig

```diff
@@ -15,6 +15,36 @@
             </p>
             <input type="text" id="input-provide-full-path" class="full-width" name="eg_config_path" value="" placeholder="optional/subdirectory"/>
         </label>
+        <label id="upload_sections">
+            <p>
+                <strong>Do you want to restrict this course material to some sections?</strong><br />
+            </p>
+            <label class = "radio">
+                <input id = "all_Sections_Showing" name="show_Some_Section_Selection" type = "radio" onchange="showDiv(0)" checked>
+                No
+            </label>
+            <label class = "radio">
+                <input name="show_Some_Section_Selection" type = "radio" onchange="showDiv(1)">
+                Yes
+            </label>
+            <script type="text/javascript">
+                function showDiv(val){
```

Comment:
Lets rename this to something more descriptive like `toggleRegistrationSections` or something 
and you can use jquery's '$('#show-Some-Section-Selection`).toggle();` instead of the if else statement

Reply:
Renamed to toggleRegistrationSections and switched to .toggle